src/ptvsd/wrapper.py

```diff
@@ -1166,28 +1162,21 @@ def _ensure_pydevd_requests_handled(self):
         #
         # This is particularly useful for those requests that do not
         # have responses (e.g. CMD_SET_BREAK).
-        return self._send_cmd_version_command()
+        pass
 
     # VSC protocol handlers
 
     @async_handler
     def on_configurationDone(self, request, args):
         self._process_debug_options(self.debug_options)
-        yield self._ensure_pydevd_requests_handled()
 
+        pydevd_request = copy.deepcopy(request)
+        del pydevd_request['seq']  # A new seq should be created for pydevd.
+        yield self.pydevd_request(-1, pydevd_request, is_json=True)
```

Comment:
This should be:

```
self._forward_request_to_pydevd(request, args, send_response=False)
```

Note that right now it should not wait on the response because it'll break tests (in tests that don't have any breakpoints, it's possible that the program starts running and finishes before this request is sent back).